cmake/Utils.cmake

```diff
@@ -278,7 +278,7 @@ function(target_enable_style_warnings TARGET)
             -Wredundant-decls
             -Wno-shadow
             -Wsign-promo
-            -Wstrict-overflow=5
+            -Wno-strict-overflow
```

Comment:
@goldsborough heads up that we're changing this. @mingzhe09088 can you give a link to the related CI failure? Thanks.

Reply:
@orionr I couldn't find the link to the related CI failure. The change was in this commit: https://github.com/pytorch/pytorch/pull/8338/commits/28e83aa637890f18cf095f93c979991560c3daaf

Reply:
Probably hard to find... In that case, @goldsborough are you okay with us temporarily changing it to get BUILD_CAFFE2 and BUILD_ATEN removal landed? We can then add it back.